cloud/deployment/fromfile/fromfile.go

```diff
@@ -33,6 +33,7 @@ var (
 	errInvalidValue                   = errors.New("is not valid")
 	errNotPermitted                   = errors.New("is not permitted")
 	canCiCdDeploy                     = deployment.CanCiCdDeploy
+	dagDeploy                         bool
```

Comment:
nit: this should be part of `CreateOrUpdate` function logic, since this variable is local to that function only

Reply:
it's used in CreateOrUpdate and getCreateOrUpdateInput

Reply:
okay fixed